doc/psyclone_script.rst

```diff
@@ -160,36 +162,37 @@ Therefore, for PSyclone to find Kernel files, the module name of a
 kernel file must be the same as its filename.  By default the parser
 looks for the kernel file in the same directory as the algorithm
 file. If this file is not found then an error is reported.
-::
 
-    > python <PSYCLONEHOME>/src/generator.py use.f90 
+.. code-block:: bash
+
+    > psyclone use.f90 
     Kernel file 'testkern.[fF]90' not found in <location>
 
-The -d option can be used to tell the generator.py script where to
-look for Kernel files. The -d option tells the generator.py script
-that the required Kernel code is somewhere within the specified
-directory hierarchy. The script will recurse from the specified
-directory path to look for the required file. There must be only one
-instance of the specified file within the specified directory:
-::
+The ``-d`` option can be used to tell ``psyclone`` where to look for
+Kernel files by supplying it with a directory. The script will recurse
+from the specified directory path to look for the required file. There
+must be only one instance of the specified file within (or below) the
+specified directory:
 
-    > cd <PSYCLONEHOME>/src
-    > python ./generator.py -d . use.f90 
+.. code-block:: bash
+		  
+    > cd <PSYCLONEHOME>/psyclone/src
+    > psyclone -d . use.f90 
     More than one match for kernel file 'testkern.[fF]90' found!
-    > python ./generator.py -d tests/test_files/dynamo0p3 -api dynamo0.3 use.f90 
+    > psyclone -d tests/test_files/dynamo0p3 -api dynamo0.3 use.f90 
     [code output]
```

Comment:
This does not produce code output. Instead, it returns an error 
```
"Parse Error: Kernel 'testkern_type' called from the algorithm layer with an insufficient number of arguments as specified by the metadata. Expected at least '5' but found '4'."
```
Please update `call invoke(testkern_type(a,b,c,d))` below `cat use_f90` and retest the example.

Reply:
Oops - well spotted. Fixed now.